src/components/Webchat.tsx

```diff
@@ -23,6 +23,8 @@ export function renderActivity(
     renderSelected: ((activity: Activity) => JSX.Element | null) | null,
     editType: EditDialogType,
     shouldRenderHighlight: boolean,
+    padding?: number,
+    hidden?: boolean,
 ): JSX.Element {
```

Comment:
I noticed you are you use `JSX.Element` instead of `React.ReactNode` and wondered if this is intentional or not? I thought `React.ReactNode` was preferred when possible.

Reply:
done